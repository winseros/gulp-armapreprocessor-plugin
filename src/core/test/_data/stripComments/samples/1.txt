some text //with inline
 // comment
 another text//also with
a quoted "//text " with ' //quotes'

// a bit more
of this
/