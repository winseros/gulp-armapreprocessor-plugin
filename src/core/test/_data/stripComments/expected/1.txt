some text 
 
 another text
a quoted "//text " with ' //quotes'


of this
/